docs/usage/configuration/config_documentation.md

```diff
@@ -2655,6 +2655,12 @@ is still supported for backwards-compatibility, but it is deprecated.
 warning on start-up. To suppress this warning, set
 `suppress_key_server_warning` to true.
 
+If the use of a trusted key server has to be deactivated, e.g. in a private
+federation or for privacy reasons, this can be realised by setting
+an empty array (`trusted_key_servers: []`). Then Synapse will request the keys
+directly from the communication partner. If Synapse does not get keys directly
+from the communication partner, the events of the other server will be rejected.
```

Comment:
I think "communication partner" is a bit unclear here. I can assume you mean the keys will be fetched directly from the servers owning them, but I think that's a bit unclear to someone who's less familiar with Synapse.

Reply:
Yes, what is a better wording? "federation partner", "server that owns the keys"? Other suggestions?

Reply:
Yes I think "server that owns the keys" is clearer.